libs/server-sdk/src/data_components/serialization_adapters/json_data_reader.hpp

```diff
@@ -1,16 +1,15 @@
 #pragma once
```

Comment:
Since `IPullSource` is now an `IDataReader`, it makes sense that the `JsonSource` is now a `JsonDataReader`.